src/components/ads/AdHandling.js

```diff
@@ -102,16 +98,16 @@ const AdHandler = () => {
             </div>
 
             {selectedAd && (
-                <div className="popup-ads fixed top-0 left-0 z-50 w-full h-full bg-black bg-opacity-80 flex items-center justify-center">
-                    <div className="relative bg-black bg-opacity-50 p-4 rounded-lg border-2 border-opacity-40 border-pink-600">
-                        <span onClick={closeModal} className="absolute top-2 right-2 text-3xl font-bold cursor-pointer">
+                <div className="p-4 fixed top-0 left-0 z-50 w-full h-full bg-black bg-opacity-80 flex items-center justify-center">
```

Comment:
changed some tailwind css to insure it looks user friendly on all types of screes (mobile, tablet and desktop)